src/components/notification/index.tsx

```diff
@@ -71,7 +72,7 @@ export class Notification {
 					""
 				)}
 				<span class={this.icon ? "icon" : "clean"}>
-					{this.icon ? <smoothly-icon name={this.properties[1]}></smoothly-icon> : ""}
+					{this.icon ? <smoothly-icon name={this.properties[1] as Icon}></smoothly-icon> : ""}
```

Comment:
change the type from `string` to `Icon` on the properties getter instead of casting. If there is no good value to replace the empty string with use the new `"empty"` or `undefined` and type it as `Icon | undefined`

Reply:
Good input, this one is solved.